### ISSUE TITLE
Unable to excute nori.

### ISSUE THREAD

**Issue by FrostedPenguin** [wjakob/nori#6]
Hello. 
I built nori successfully in Ubuntu 16.04.
However when I try to run it with example scene, it throws error like this.

> Fatal error: Error while parsing "../scenes/pa2/ajax-normals.xml": A constructor for class "normals" could not be found! (at line 8, col 6)

I tried both of scenes in scenes folder and both of them gives me same error.

So I open ajax-normals.xml file and there is only one line has type "normals".

`
<integrator type="normals"/>`

So does it means should I implement integrator for normals?
Or is there anything else that I'm missing?

**Comment by FrostedPenguin** [comment]
I should have checked nori-old repository first.

At their I found [assignment specs](https://wjakob.github.io/nori-old/) and it turns out that I should implement integrator for type "normals". So I decided to follow instructions and implement some codes one by one..
Probably nori will be good starting point to start studying ray tracer.

I was quite shocked by the fact that there is a global illumination class in university... Super envy.
It is really hard to find class like this in my country since a few of people study global illumination. So I have to study from scratch by my self.
